go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/filecoin-project/go-amt-ipld/v3
 go 1.12
 
 require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/ipfs/go-block-format v0.0.2
 	github.com/ipfs/go-cid v0.0.7
 	github.com/ipfs/go-ipld-cbor v0.0.4
-	github.com/ipfs/go-log v1.0.4
+	github.com/kr/pretty v0.1.0 // indirect
```

Comment:
why update these?

Reply:
I didn't intend to, I just ran `go mod tidy` before I pushed. I think they were dirty from a past commit.